[2022-07-19 12:08:21,234] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T12:07:00+00:00 [queued]>
[2022-07-19 12:08:21,295] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: simple_dag.respond scheduled__2022-07-19T12:07:00+00:00 [queued]>
[2022-07-19 12:08:21,296] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 12:08:21,298] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-19 12:08:21,372] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-19 12:08:21,445] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): respond> on 2022-07-19 12:07:00+00:00
[2022-07-19 12:08:21,461] {standard_task_runner.py:52} INFO - Started process 13410 to run task
[2022-07-19 12:08:21,478] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'simple_dag', 'respond', 'scheduled__2022-07-19T12:07:00+00:00', '--job-id', '703', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp82ilzucx', '--error-file', '/tmp/tmp1lagewkh']
[2022-07-19 12:08:21,483] {standard_task_runner.py:80} INFO - Job 703: Subtask respond
[2022-07-19 12:08:21,847] {task_command.py:370} INFO - Running <TaskInstance: simple_dag.respond scheduled__2022-07-19T12:07:00+00:00 [running]> on host 02ffb48252da
[2022-07-19 12:08:22,369] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=simple_dag
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2022-07-19T12:07:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-19T12:07:00+00:00
[2022-07-19 12:08:44,047] {logging_mixin.py:115} INFO - (2724, 1)
[2022-07-19 12:08:44,052] {python.py:173} INFO - Done. Returned value was: Greet Responded Again
[2022-07-19 12:08:44,080] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=simple_dag, task_id=respond, execution_date=20220719T120700, start_date=20220719T120821, end_date=20220719T120844
[2022-07-19 12:08:44,118] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-19 12:08:44,143] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
